verifier/verifier.go

```diff
@@ -1082,3 +1043,43 @@ func verifyTimestamp(ctx context.Context, policyName string, trustStores []strin
 	// success
 	return nil
 }
+
+// setRevocation sets revocation validators of verifier
+func setRevocation(verifier *verifier, verifierOptions VerifierOptions) error {
```

Comment:
Previously, I was thinking of something like
```go
func getRevocationTimestampingValidator(verifierOptions VerifierOptions) (Validator, error) {
```

However, with your current code structure, it makes more sense if you change it back to
```go
func (v *verifier) setRevocation(verifierOptions VerifierOptions) error {
```

Reply:
Updated.